packages/utils/src/fallbackSubgraph.ts

```diff
@@ -0,0 +1,80 @@
+import { Logger } from "./logger";
+import { createLoggingContext } from "./request";
+
+export type SubgraphSyncRecord = {
+  synced: boolean;
+  latestBlock: number;
+  syncedBlock: number;
+}
+
+type Sdk<T> = {
+  client: T;
+  record: SubgraphSyncRecord;
+};
+
+export class FallbackSubgraph<T extends { GetBlockNumber: () => Promise<any>; }> {
```

Comment:
Should really define the type it's extending, since it should also be used in the `Sdk<T>` type above

Reply:
Ahhh gotcha, will do